engine/orchestration/src/main/java/com/cloud/agent/manager/AgentManagerImpl.java

```diff
@@ -122,7 +122,6 @@
  **/
 public class AgentManagerImpl extends ManagerBase implements AgentManager, HandlerFactory, Configurable {
     protected static final Logger s_logger = Logger.getLogger(AgentManagerImpl.class);
-    protected static final Logger status_logger = Logger.getLogger(Status.class);
```

Comment:
this seems a functional backwards incompatible change. Did you considder that?

Reply:
Yes, I did. It is a change in the logging class that is used on the left side of the log message in the log file. People normally lock down their log parse in the message per se, but for sure if they are considering the class as well, this can break things.
I changed this for a simple reason. It is misleading. It will show as if the “Status” class is generating that message, which is not true. I see no reason for this type of code.

I can revert this change, but my position is that it does not make sense. It is not used consistently. The same type of message is being logged also with the other logger.

Reply:
I agree it is a corner case and we can revert that if someone complains but it is an interface of the system, through the log configuration.
I am fine with it, juat wanted to be sure you realised.